frontend/src/components/translation/FontSizeSlider.css

```diff
@@ -0,0 +1,21 @@
+.slider-container{
```

Comment:
With chakra ui we can avoid making new css files, here are some options
- style components directly - good for little styling; `<Box width="100px">`
- make a new style variant (examples https://chakra-ui.com/docs/theming/component-style)